plugins/techdocs/src/home/components/OwnedContent.tsx

```diff
@@ -0,0 +1,133 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { useCopyToClipboard } from 'react-use';
+import { generatePath } from 'react-router-dom';
+
+import { IconButton, Tooltip } from '@material-ui/core';
+import ShareIcon from '@material-ui/icons/Share';
+import {
+  Content,
+  ContentHeader,
+  SupportButton,
+  Table,
+  identityApiRef,
+  useApi,
+  EmptyState,
+  Button,
+  SubvalueCell,
+  Link,
+} from '@backstage/core';
+import { Entity } from '@backstage/catalog-model';
+
+import { rootDocsRouteRef } from '../../plugin';
+
+export const OwnedContent = ({ value }: { value: Entity[] | undefined }) => {
+  const [, copyToClipboard] = useCopyToClipboard();
+  const identityApi = useApi(identityApiRef);
+  const userId = identityApi.getUserId();
+
+  if (!value || !userId) return null;
+
+  const ownedDocuments = value
+    .filter((entity: Entity) => entity?.spec?.owner === userId)
+    .map(entity => {
+      return {
+        name: entity.metadata.name,
+        description: entity.metadata.description,
+        owner: entity?.spec?.owner,
+        type: entity?.spec?.type,
+        docsUrl: generatePath(rootDocsRouteRef.path, {
+          namespace: entity.metadata.namespace ?? 'default',
+          kind: entity.kind,
+          name: entity.metadata.name,
+        }),
+      };
+    });
+
+  const columns = [
+    {
+      title: 'Document',
+      field: 'name',
+      highlight: true,
+      render: (row: any): React.ReactNode => (
+        <SubvalueCell
+          value={<Link to={row.docsUrl}>{row.name}</Link>}
+          subvalue={row.description}
+        />
+      ),
+    },
+    {
+      title: 'Owner',
+      field: 'owner',
+    },
+    {
+      title: 'Type',
+      field: 'type',
+    },
+    {
+      title: 'Actions',
+      width: '10%',
+      render: (row: any) => (
+        <Tooltip title="Click to copy documentation link to clipboard">
+          <IconButton
+            onClick={() =>
+              copyToClipboard(`${window.location.origin}/${row.docsUrl}`)
+            }
+          >
+            <ShareIcon />
+          </IconButton>
+        </Tooltip>
+      ),
+    },
+  ];
+
+  return (
+    <Content>
+      <ContentHeader
+        title="Owned documents"
+        description="Access your documentation."
+      >
+        <SupportButton>Discover documentation you own.</SupportButton>
+      </ContentHeader>
+      {ownedDocuments && ownedDocuments.length > 0 ? (
+        <Table
+          options={{ paging: true, pageSize: 20, search: false }}
```

Comment:
I think the search might be useful to enable here?

Reply:
Will combine my answer to this with the general feedback above. Haven't spent any time in changing the "Overview" page right now (except from adding a title) because yes, it will include a full search experience when search is ready. You can see more designs of that in this [issue](https://github.com/backstage/backstage/issues/2028). So we just left that out for now and focused on the owned page. With that said, i'll enable search on the table for now as it makes sense until we have a full search page. Thanks for pointing that out!